### ISSUE TITLE
Loss function and metrics

### ISSUE THREAD

**Issue by cyhuangx** [camlab-ethz/poseidon#7]
In equation (C.1) in the paper, the training loss is calculated as the average of the point-wise relative L1 losses, but in your implemetation scOT/metrics.py, the relative Lp error (per sample) is calculated as the ratio of the integral of the error and the integral of the truth.

Does this mean that different metrics are used during training and evaluation? If this is the case, is there a special reason for these choices?

**Comment by maximilianherde** [comment]
No, the same metrics are used during training and evaluation, this is a typo in the appendix of the paper; it should be a realisation of the standard relative L^1 loss (with epsilon in the denominator).

For the loss, see scOT/model.py (the end of the model forward includes the loss calculation), for eval and test, see scOT/metrics.py.
